src/frontend/src/lib/components/tokens/TokensSignedIn.svelte

```diff
@@ -21,10 +22,38 @@
 		({ id: tokenId }) =>
 			($balancesStore?.[tokenId]?.data ?? BigNumber.from(0n)).gt(0n) || displayZeroBalance
 	);
+
+	let sortingEnabled = true;
+	$: $pointerEventStore, (sortingEnabled = !$pointerEventStore);
+
+	const updateTokensToDisplay = ({
+		tokensToDisplay,
+		newTokensList
+	}: {
+		tokensToDisplay: Token[];
+		newTokensList: Token[];
+	}): Token[] => {
+		if (sortingEnabled) {
```

Comment:
At startup that function runs 22 times and I don't even have done anything yet. This feels really unperformant.

<img width="1536" alt="Capture d’écran 2024-08-21 à 08 20 42" src="https://github.com/user-attachments/assets/14180d61-2484-4bc3-b5a8-173c6918c301">

Reply:
I refactored a bit to simplify the logic, and included some comments there. The functions runs indeed if there are no pointer events, but it would just return the normal list given by the store. So it would refresh with the store refreshing.

The logic is as follows:

- if there is no pointer event, just return the normal token list given by the store (that is refreshed quite a lot at login because it depends on fetching some data like `balances` and `exchanges`)
- if there are pointes events:
  - if the order of the list showed IS the same as the one that is being updated by the store, then just return the normal list given by the store
  - if the order of the list showed IS NOT the same as the one that is being updated, then keep the order showed and just update the balances
  
 Below two practical examples (see console logs in images):
 
 ### The page loads and the user does not trigger pointer events
 The function is running but just returning the list from the store, as usual. So it runs every time the store changes
 
<img width="1503" alt="Screenshot 2024-08-22 at 08 35 11" src="https://github.com/user-attachments/assets/a5781a0c-72f6-4e35-8984-d2ac4aab2ef9">


 ### The page loads and the user triggers pointer events while waiting
 The function is running but it keeps the same order showed before the pointer event, however it is updating the balances.
 
<img width="1511" alt="Screenshot 2024-08-22 at 08 35 34" src="https://github.com/user-attachments/assets/38a3c636-72af-4697-9dac-859dc463e3f2">

 
 # NOTE
 We can think of optimizing the refresh of the store of the token list in another PR, to avoid showing the list at all at login, unless every fetching is complete.